src/main/kotlin/omok/model/RenjuRuleAdaptor.kt

```diff
@@ -0,0 +1,29 @@
+package omok.model
+
+import omok.rule.RenjuRule
+
+object RenjuRuleAdaptor {
+    fun isForbidden(
+        board: Board,
+        stone: Stone,
+    ): Boolean {
+        val boardConverted = board.convert()
+        val point = stone.point
+        val rule = RenjuRule(boardConverted, boardSize = 15)
+        return rule.run {
+            checkThreeThree(point.x, point.y) || checkFourFour(point.x, point.y) || checkMoreThanFive(point.x, point.y)
+        }
```

Comment:
rule은 사실 어댑터 인스턴스가 생성될 때 이미 정해져있지 않을까요?
```suggestion
class RenjuRuleAdapter(rule: OmokRule) {
    ....
    fun isForbidden(...): Boolean {
        return  rule.validPosition(board, point.x, point.y)
     }
}
        
```